include/dt-bindings/pinctrl/stm32-pinctrl.h

```diff
@@ -53,29 +71,6 @@
 #define STM32_PUPDR_MASK		0x3
 #define STM32_PUPDR_SHIFT		9
 
-/* Usefull definitions */
-#define STM32_PUSHPULL_NOPULL    (STM32_OTYPER_PUSH_PULL | STM32_PUPDR_NO_PULL)
-#define STM32_OPENDRAIN_PULLUP   (STM32_OTYPER_OPEN_DRAIN | STM32_PUPDR_PULL_UP)
-#define STM32_PUSHPULL_PULLUP	 (STM32_OTYPER_PUSH_PULL | STM32_PUPDR_PULL_UP)
-
-/* Alternate functions numbers*/
```

Comment:
Actually, I just moved them up into /* Alternate functions numbers */ section for coherency